test/bundling.ts

```diff
@@ -888,13 +907,13 @@ describe('bundleable', () => {
       expect(obj[0].field1).to.deep.equal([1, 2, 3]);
     }
     const apiCall = createApiCall(spy, settings);
-    apiCall({field1: [1, 2, 3], field2: 'id'}, null)
+    apiCall({field1: [1, 2, 3], field2: 'id'}, undefined)
```

Comment:
This is indeed typing related, but the underlying reasoning is simple: `apiCall` accepts `(argument: {}, callOptions?: CallOptions, callback?: APICallback)`, so no options would be automatically mapped to `undefined`, so `null` is not welcome here.

In general, I agree with the post you linked about not using `null` at all. One place where it still exists in our code is when we call a callback: `callback(null, result)` vs `callback(err, null)`. It certainly can be changed to `undefined` but I would consider this to be a hidden breaking change (someone might check `if (err === null)` in their code and it will stop working properly), so I'd probably leave those nulls there for the time being.